hdgi/.htaccess

```diff
@@ -43,6 +43,15 @@ RewriteRule ^$ https://madhawap.github.io/human-device-gesture-interaction-ontol
 RewriteCond %{HTTP_ACCEPT} .+
 RewriteRule ^$ https://madhawap.github.io/human-device-gesture-interaction-ontology/v0.1/406.html [R=406,L]
 
+# Redirect https://w3id.org/hdgi/v0.1 to https://w3id.org/hdgi
+Redirect /hdgi/v0.1 https://w3id.org/hdgi
```

Comment:
These are relative to the current `hdgi` directory, so you don't want the `/hgdi/` prefix.  Usually should be something more strict like `^v0.1$`.  Similar below.

Reply:
Changed Redirect to RewriteRule with regex and tested locally.